common/selectors/transaction/meta.ts

```diff
@@ -11,8 +11,8 @@ const getTokenValue = (state: AppState) => getMetaState(state).tokenValue;
 const getUnit = (state: AppState) => getMetaState(state).unit;
 const getPreviousUnit = (state: AppState) => getMetaState(state).previousUnit;
 const getDecimalFromUnit = (state: AppState, unit: string) => {
-  if (isEtherUnit(unit)) {
-    return getDecimalFromEtherUnit('ether');
+  if (isNetworkUnit(state, unit)) {
+    return getDecimal(state);
```

Comment:
This should be getting the decimal from the current unit passed instead, we dont have defined decimals for now (see https://github.com/MyCryptoHQ/MyCrypto/issues/1185) but, lets stick to `getDecimalFromEtherUnit('ether');` in the meanwhile